flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -159,8 +159,10 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
 
     // Clearing the display tracker here is safe since didPush happens before the Widget is built
     _timeToDisplayTracker?.clear();
-    _finishTimeToDisplayTracking();
-    _startTimeToDisplayTracking(route);
+
+    DateTime timestamp = _hub.options.clock();
+    _finishTimeToDisplayTracking(endTimestamp: timestamp);
+    _startTimeToDisplayTracking(route, timestamp);
```

Comment:
why don't we put them both in an async function with await? or does that modify the desired behaviour.

Reply:
The navigator tests were broken because of it. Also, this did not fix the problem, so it's ok to revert.